components/remote_settings/src/remotesettings.udl

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+namespace remotesettings {};
+
+[Custom]
+typedef string JsonObject;
+
+dictionary RemoteSettingsConfig {
+    string? server_url;
+    string? bucket_name;
+    string collection_name;
+};
+
+[Error]
+enum RemoteSettingsError {
+    "NetworkFailure",
```

Comment:
nano-nit - one being `Failure` and the rest being `Error` seems odd?

Reply:
And it wasn't even used! Think it was leftover from when I was stubbing everything out